web/unelte/vagrant/nginx/Vagrantfile

```diff
@@ -0,0 +1,33 @@
+api_version = '2'
+provider    = 'virtualbox'
+box         = 'hashicorp/precise64'
+hostname    = 'ngix'
+ip_address  = '192.168.50.3'
```

Comment:
I would change it to `192.168.50.200`. When you have a hardcoded ip, it's better to have it as a higher value so that a DHCP won't interfere with it.